### PR TITLE
test: skip unsupported features tests

### PR DESCRIPTION
**Pull request by IlyaFaer**
Spanner doesn't support temporary tables. The same tests (for indexes, table names, etc.) are passing on real tables, so these tests can be skipped.

### CODE REVIEW

File: test/test_suite.py [hunk 1]
```diff
@@ -682,3 +685,17 @@ def test_unique_constraint_raises(self):
...
+
+    @pytest.mark.skip("Spanner doesn't support temporary tables")
+    def test_get_temp_table_unique_constraints(self):
+        pass
+
+
+class InsertBehaviorTest(_InsertBehaviorTest):
+    @pytest.mark.skip("Spanner doesn't support empty inserts")
+    def test_empty_insert(self):
+        pass
```
Comment (IlyaFaer) [inline test/test_suite.py:701]: Also skippting a test with an empty insert. See [the docs](https://cloud.google.com/spanner/docs/dml-syntax#insert-statement).

File: google/cloud/sqlalchemy_spanner/requirements.py [hunk 2]
```diff
@@ -37,6 +37,10 @@ def autocommit(self):
     def order_by_collation(self):
         return exclusions.open()
 
+    @property
+    def implements_get_lastrowid(self):
+        return exclusions.closed()
```
Comment (IlyaFaer) [inline google/cloud/sqlalchemy_spanner/requirements.py:42]: `lastrowid` is not supported

**Review by AVaksman — APPROVED** [review]
